build.gradle

```diff
@@ -0,0 +1,132 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url "https://plugins.gradle.org/m2/"
+        }
+    }
+    dependencies {
+        classpath 'com.google.gradle:osdetector-gradle-plugin:1.4.0'
+        classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.9'
+    }
+}
+
+subprojects {
+    apply plugin: "checkstyle"
+    apply plugin: 'maven'
+    apply plugin: 'idea'
+    apply plugin: 'java'
+    apply plugin: "signing"
+    apply plugin: "jacoco"
+    // TODO(bdrutu): enable all checks.
+    apply plugin: "net.ltgt.errorprone"
+
+    group = "com.google.instrumentation"
+    version = "0.1.0-SNAPSHOT" // CURRENT_INSTRUMENTATION_VERSION
```

Comment:
The latest release is 0.3.0. It should be either 0.3.1-SNAPSHOT or 0.4.0-SNAPSHOT.